src/Employer/Employer.Web/Views/ApplicationReview/ApplicationReview.cshtml

```diff
@@ -253,21 +225,18 @@
             }
 
         </section>
-        <section>
-            <div class="grid-row">
-                <div class="column-two-thirds">
-                    <h2 class="heading-large">Skills the Employer wants</h2>
-                </div>
 
-            </div>
+        <section>
+            <h2 class="heading-large">Skills the Employer wants</h2>
 
             <p asp-condition="@Model.HasNoSkills">I don't have the skills the employer wants</p>
 
             <p asp-condition="@Model.HasSkills">
                 @Html.Raw(string.Join(", ", Model.Skills.Select(s => string.Format("<span>{0}</span>", Html.Encode(s)))))

```

Comment:
Do we have to surround each skill in a `span`? 👍 for the encoding, I think we should do this on the preview views as well.

Reply:
👍